geode-core/src/main/java/org/apache/geode/internal/cache/tier/sockets/ServerConnection.java

```diff
@@ -1412,11 +1412,19 @@ public boolean cleanup() {
       getAcceptor().decClientServerConnectionCount();
     }
 
-    try {
-      theSocket.close();
-    } catch (Exception ignored) {
-    }
+    if (!theSocket.isClosed()) {
+      final String closerName =
+          communicationMode.isWAN() ? "WANSocketCloser" : "CacheServerSocketCloser";
```

Comment:
`closerName` is being passed as the `address` parameter of `asyncClose()`. The doc on that method says:

```
   * @param address identifies who the socket is connected to
```

Other calls to `asyncClose()` send an address. If there is a reason we don't want to send an address here, would you mind adding a comment to explain it?

Reply:
comments added.  I don't want cache shutdown to use a different executor for each remote address.  There could be hundreds of them.